index.html

```diff
@@ -438,6 +438,16 @@ <h2>
         />
       </a>
     </footer>
+    <div class="menu-bar">
+      <i class="fa fa-times"></i>
+      <div class="header-slide-links">
+        <li><a >Portifolio</a></li>
+        <li><a >About</a></li>
+        <!-- <li><a >Contact</a></li> -->
+        <li><a href="#contact">Contact</a></li>
```

Comment:
- [ ] I suggest that you avoid leaving commented pieces of code in your codebase as this clutters your codebase and makes it less readable.